src/Sunny.jl

```diff
@@ -101,9 +98,18 @@ include("SunnyGfx/SunnyGfx.jl")
 include("SunnyGfx/CrystalViewer.jl")
 export view_crystal, offline_viewers, browser
 
-include("ParallelTempering.jl")
+include("MonteCarlo/Samplers.jl")
+export propose_uniform, propose_flip, propose_delta, @mix_proposals, LocalSampler
+
+include("MonteCarlo/BinnedArray.jl")
+export BinnedArray, reset!, get_keys, get_vals, get_pairs
```

Comment:
Consider removing these exports if they are not needed by users of Sunny (it creates a backward incompatibility danger).

Reply:
I was intending for these exports to be called by users so they can create histograms and collect statistics for measured observables with PT. 

I decided not to make a BinnedArray directly within PT so that the users are not forced to bin measurements. I may need to implement one more convenience function for using PT with Langevin so that the systems are returned in order of increasing temperature. Right now, you would need to do PT.systems[PT.system_ids] to get the correct order and then calculate measurements.